Rubberduck.Parsing/ComReflection/SerializableProjectBuilder.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.VBEditor;
+using Rubberduck.VBEditor.Extensions;
+
+namespace Rubberduck.Parsing.ComReflection
+{
+    public class SerializableProjectBuilder : ISerializableProjectBuilder
```

Comment:
Do we really expose serializing declaration through the API atm? We definitly no longer load them through the API.

Reply:
We really have to expose the `SerializableDeclarationsBuilder` in `Parsing` and cannot put it into the test project because it is required for the `SerializeCommend` that has to live in `Core`. (We need to generate the serialized declarations from somewhere.) 

The `SerializedReferencedDeclarationsCollector` on the other hand could be moved to the test project by now. So, if you want me to move that, please let me know.

Reply:
In that case, I would not bother. I probably would favor keeping related classes together than scattered across projects.